CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
@bascht: Ignore my comment ... just saw that there *are* instructions already 🙈 

@DreamFlasher: The ugly about this is that you commit stuff already before you have tested it locally ... but that may be worth it to bypass the local setup 😸